source/qml_v2/AdaptiveCardQmlEngine/models/CollectionItemModel.h

```diff
@@ -2,16 +2,18 @@
 
 #include <QAbstractListModel>
 #include <SharedAdaptiveCard.h>
-
 #include <TextBlock.h>
 #include <RichTextBlock.h>
 #include <Image.h>
 #include <TextInput.h>
+#include "RichTextBlock.h"
+#include <NumberInput.h>
 #include "Enums.h"
```

Comment:
We should ideally use double quotes for include  here, as these files are part of our project and not external library.
Please keep a note of this, will clean it up in a separate PR.